charts/opentelemetry-demo/examples/bring-your-own-observability/rendered/component.yaml

```
@@ -1575,15 +1579,15 @@ spec:
           - name: LOCUST_SPAWN_RATE
             value: "1"
           - name: LOCUST_HOST
-            value: http://example-frontend:8080
+            value: http://example-frontendproxy:8080
           - name: LOCUST_HEADLESS
             value: "false"
           - name: LOCUST_AUTOSTART
             value: "true"
           - name: PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION
             value: python
-          - name: OTEL_EXPORTER_OTLP_TRACES_ENDPOINT
-            value: http://$(OTEL_COLLECTOR_NAME):4318/v1/traces
+          - name: OTEL_EXPORTER_OTLP_ENDPOINT
+            value: http://$(OTEL_COLLECTOR_NAME):4317
```

This is the url for Front-End to call to Front-End proxy
```suggestion
            value: http://$(OTEL_COLLECTOR_NAME):4317/oltp-http/v1/traces
```